Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -30,14 +29,15 @@ public class PowerShellKernel : KernelBase
 
         private readonly PSKernelHost _psHost;
         private readonly Lazy<PowerShell> _lazyPwsh;
+        private PowerShell Pwsh => _lazyPwsh.Value;
```

Comment:
nipick: what do you think if we name this private property `pwsh`?

Reply:
I'm cool with that.